Sources/TransferMethod/SelectTransferMethodTypePresenter.swift

```diff
@@ -141,11 +144,19 @@ final class SelectTransferMethodTypePresenter {
 
     /// Navigate to AddTransferMethodController
     func navigateToAddTransferMethod(_ index: Int) {
-        if let profileType = user?.profileType?.rawValue {
-            view.navigateToAddTransferMethodController(country: selectedCountry,
-                                                       currency: selectedCurrency,
-                                                       profileType: profileType,
-                                                       transferMethodTypeCode: sectionData[index].code!)
+        userRepository.getUser {[weak self] (getUserResult) in
```

Comment:
Since the `user` is in the cache, Can you remove this `weak self` ?

Reply:
My understanding is that closure captures context, so we should always weakify`self` to avoid retain cycles